src/sql/src/plan/statement/ddl.rs

```diff
@@ -2655,16 +2655,18 @@ impl KafkaConnectionOptionExtracted {
             (None, Some(v)) => v.to_vec(),
         };
 
+        // TODO: assert that all brokers are configured the same, instead of mixed and matched.
```

Comment:
```suggestion
```

Either we should do this now, or never. (It's a backincompat change to do later.) But tbqh "never" sounds good to me.

Reply:
Yeah, "never" seems fine to me to; when we extend this and allow top-level `SSH TUNNEL`, we can assert that there are no `USING`'s at all